### DISCUSSION TITLE
Generating questions (rather than answers) from embeddings

### DISCUSSION THREAD

**Discussion by Shivak11** [langflow-ai/langflow#425] [Q&A]
I want to embed a book but instead of answering my questions, I want the LLM to ask me a question based on the embedded book. Is it possible? I think I'll need to prompt this somewhere but there is no provision to set a prompt for a vectorstoreagent. Can anyone guide me on this? Thanks!

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
